.github/ci/unbottled_dependencies.bash

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+# Copyright (C) 2024 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# The script will print the unbottled dependencies from the osrf/simulation
+# tap of a specified formula.
+#
+# Usage:
+# $ ./unbottled_dependencies.bash <formula>
+#
+# For example, to print the unbottled dependencies of gz-harmonic from the
+# osrf/simulation tap:
+#
+# ./unbottled_dependencies.bash gz-harmonic
+
+FORMULA=${1}
+
+if [[ $# -ne 1 ]]; then
```

Comment:
```suggestion
if [ $# -ne 1 ]; then
```

This can use `test` / `[]` instead of the bash conditional expression `[[]]``.

Reply:
switched to `[]` in https://github.com/osrf/homebrew-simulation/pull/2610/commits/d45f9e64382fc869d5834aae39c621d193087d6c